### PR TITLE
Create AniketNS.mdx

### PR DESCRIPTION
**Pull request by AniketNS**
## Fixes Issue

Closes #<issue_number>

### CODE REVIEW

File: contribution/AniketNS.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        </svg>
+      }
+      level="Newbie" 
+    >
+        your experience
```
Comment (priyankarpal) [inline contribution/AniketNS.mdx:31]: add your experience in line `31`

File: contribution/AniketNS.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    <Skill
+      name="skill name"
+      href="official website"
+      level="Intermidiate"
+      icon={
+      add a svg icon of your skills
+      }
+    >
+   your experience
+    </Skill>
```
Comment (priyankarpal) [inline contribution/AniketNS.mdx:42]: add your details such as skills name, experience etc.

File: contribution/AniketNS.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+## Hobbies
+
+- Reading
+
+<Projects>
+  <Project
+    name="Ration Shop Management System"
+    repository= https://github.com/AniketNS/Ration-Shop-management
+   >
+ add a description
```
Comment (priyankarpal) [inline contribution/AniketNS.mdx:82]: add a description

File: contribution/AniketNS.mdx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      add a svg icon of your skills
+      }
+    >
+   your experience
+    </Skill>
+
+  </Skills>
+</Tab>
+
+</Tabs>
```
Comment (priyankarpal) [inline contribution/AniketNS.mdx:47]: remove this line. 

```suggestion

```

File: contribution/AniketNS.mdx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24">
+  <path d="M12 1.5c-6.351 0-11.5 5.149-11.5 11.5s5.149 11.5 11.5 11.5 11.5-5.149 11.5-11.5-5.149-11.5-11.5-11.5zm0 1.5c5.79 0 10.5 4.71 10.5 10.5s-4.71 10.5-10.5 10.5-10.5-4.71-10.5-10.5 4.71-10.5 10.5-10.5zm-2.25 7.5h-1.5v6h1.5v-6zm5.25 0h-1.5v6h1.5v-6z"/>
+</svg>
+}
+    </Skill>
+
+  </Skills>
+</Tab>
+
```
Comment (priyankarpal) [inline contribution/AniketNS.mdx:72]: Add `</Tabs>`

```suggestion
   </Tabs>
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: contribution/AniketNS.mdx (hunk 2 above)
Comment (AniketNS) [reply contribution/AniketNS.mdx:42]: I don't have any experience. Like I'm a college student.

File: contribution/AniketNS.mdx (hunk 3 above)
Comment (AniketNS) [reply contribution/AniketNS.mdx:82]: yup I'll do that.

File: contribution/AniketNS.mdx (hunk 4 above)
Comment (AniketNS) [reply contribution/AniketNS.mdx:47]: Ok

File: contribution/AniketNS.mdx (hunk 2 above)
Comment (priyankarpal) [reply contribution/AniketNS.mdx:42]: > I don't have any experience. Like I'm a college student.

then leave it blank or add `learning`

Comment (AniketNS) [reply contribution/AniketNS.mdx]: yup, I've removed it. Is that Ok?

File: contribution/AniketNS.mdx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+</Tabs>
+## Hobbies
+
+- Reading
+
+<Projects>
+  <Project
+    name="Ration Shop Management System"
+    description="It is simple BCA level windows desktop application. Which is based on normal ration shop system. I'd love to have your contributions and new ideas on this project."
+    repository= https://github.com/AniketNS/Ration-Shop-management
```
Comment (priyankarpal) [inline contribution/AniketNS.mdx:67]: add commas.

```suggestion
    repository= "https://github.com/AniketNS/Ration-Shop-management"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by AniketNS** [comment]
Hey @priyankarpal, tell me if there is something I should correct.

**Comment by priyankarpal** [comment]
> Hey @priyankarpal, tell me if there is something I should correct.

looks good now. just change what I mentioned.

**Comment by AniketNS** [comment]
I've changed all that you had mentioned.

File: contribution/AniketNS.mdx (hunk 6 above)
Comment (priyankarpal) [reply contribution/AniketNS.mdx:67]: > add commas.

fix this

Comment (AniketNS) [reply contribution/AniketNS.mdx:67]: ok

**Review by priyankarpal — DISMISSED** [review]
Looks Good 🎉

**Comment by AniketNS** [comment]
when will be my PR get merged?

**Comment by AniketNS** [comment]
Ohh ok

**Comment by priyankarpal** [comment]
> Can I have your twitter Id @priyankarpal ?

yeah sure. https://twitter.com/Priyankarpal

**Comment by sboy99** [comment]
Some fixes needed I will double check after some time

**Comment by priyankarpal** [comment]
> Some fixes needed I will double check after some time

not showing after deploying

**Comment by AniketNS** [comment]
Thank you so much @priyankarpal  and @sboy99

**Pull request merged**